### PR TITLE
Update some maven plugin's versions and copyright year 

### PR DESCRIPTION
**Pull request by bmscomp**
Updating some maven plugins, and copyright year 

Update some maven plugin's versions, and copyright year, and adjust some working on java doc

### CODE REVIEW

File: src/main/java/jakarta/inject/package-info.java [hunk 1]
```diff
@@ -132,7 +132,7 @@
  * product of the number of unit tests and the number of dependencies.
  *
  * <p><b>This package provides dependency injection annotations that enable
- * portable classes</b>, but it leaves external dependency configuration up to
+ * portable classes</b>, But it leaves external dependency configuration up to
```
Comment (Ladicek) [inline src/main/java/jakarta/inject/package-info.java]: This is wrong, please remove.

File: src/main/java/jakarta/inject/package-info.java [hunk 2]
```diff
@@ -146,7 +146,7 @@
  * or even plain Java code. An injector could rely on reflection or code
  * generation. An injector that uses compile-time code generation may not even
  * have its own run time representation. Other injectors may not be able to
- * generate code at all, neither at compile nor run time. A "container", for
+ * generate code at all, neither at compilation nor runtime. A "container", for
```
Comment (Ladicek) [inline src/main/java/jakarta/inject/package-info.java]: This is unnecessary. "At compile time" means "during compilation". "Run time" is probably less common than "runtime", but they do mean the same thing (plus they have other meanings).

File: src/main/java/jakarta/inject/package-info.java (hunk 1 above)
Comment (bmscomp) [reply src/main/java/jakarta/inject/package-info.java]: Yes It's a typo on my side i will remove it for sure

File: src/main/java/jakarta/inject/package-info.java (hunk 2 above)
Comment (bmscomp) [reply src/main/java/jakarta/inject/package-info.java]: Thank you for review @Ladicek , I fixed it  on the last commit

File: src/main/java/jakarta/inject/package-info.java (hunk 1 above)
Comment (bmscomp) [reply src/main/java/jakarta/inject/package-info.java]: @Ladicek   Ladicek Fixed on the last commit

**Comment by bmscomp** [comment]
@Ladicek  Pull request is updated to fix review's remarks

**Review by Ladicek — APPROVED** [review]

**Comment by Ladicek** [comment]
I'll wait for a few days so that other committers may also chime in.

**Review by Emily-Jiang — APPROVED** [review]
LGTM

**Review by manovotn — APPROVED** [review]

**Pull request merged**